components/molecules/NivoScatterChart/nivo-scatter-chart.tsx

```diff
@@ -74,15 +74,15 @@ const NivoScatterPlot = ({
   const handleTogglePrFilter = (val: string) => {
     switch (val) {
       case "0":
-        handleSetPrFilter("all");
+        handleSetPrFilter?.("all");
```

Comment:
I don't have the full context, but curious why `metaData` and `handleSetPrFilter` are optional now?

Reply:
On the `Explore` page, we use them to sort between `All`, `Open` and `Closed` pull requests. 

![image](https://github.com/open-sauced/app/assets/42211/f0f52096-c0e7-43cb-965a-332b51ecc68b)

We are only showing distribution of contributors here